diffcp/cones.py

```diff
@@ -132,7 +136,7 @@ def pi(x, cones, dual=False):
         for dim in sz:
             if cone == PSD:
                 dim = vec_psd_dim(dim)
-            elif cone == EXP:
+            elif cone == EXP or cone == EXP_DUAL:
```

Comment:
The second check (`cone == EXP_DUAL`) is unnecessary, since you overwrite cone above

Reply:
I actually need this, since I change EXP_DUAL to EXP inside `_proj`.

Reply:
op sorry, github collapsed all the code in between and I thought that lines 82-85 were a part of this function.